test/testconstructors.cpp

```diff
@@ -1952,7 +1952,8 @@ class TestConstructors : public TestFixture {
 
     void initvar_smartptr() { // #10237
         Settings s;
-        s.libraries.emplace_back("std");
+        // TODO: test shuld probably not pass without library
```

Comment:
@chrchr-github Please have a look at this. It was added in 9d6e5c2a05553f2676454febd41ea3b69ec1c36d.

Reply:
That should probably be `LOAD_LIB_2(s.library, "std.cfg");` Since the fix was for a FP, the test is ineffective, but still passes with or without the library